test/extended/util/client.go

```diff
@@ -179,6 +179,32 @@ func NewCLIWithoutNamespace(project string) *CLI {
 	return cli
 }
 
+// NewCLIForMonitorTest initializes the CLI and Kube framework helpers
+// without a namespace. Should be called outside of a Ginkgo .It()
+// function.
+func NewCLIForMonitorTest(project string) *CLI {
```

Comment:
So you needed new non ginko versions of this and Run to support running in the monitor test environment?

Thanks for setting those up.

Reply:
Correct